route/build/cpl/RtmMod.F90

```diff
@@ -413,19 +423,19 @@ SUBROUTINE route_run(rstwr)
     rtmCTL%qirrig_actual = -1._r8* rtmCTL%qirrig  ! actual water take [mm/s] - positive (take) or negative (inject)
     do nr = rtmCTL%begr,rtmCTL%endr
       ! calculate depth of irrigation [mm] during timestep
-      irrig_depth = rtmCTL%qirrig_actual(nr)* coupling_period
+      irrig_depth = rtmCTL%qirrig_actual(nr)* delt_coupling
```

Comment:
What is the difference between coupling_period and delt_coupling?

Reply:
this is coupling_period in sec (coupling_period is given in day).